### PR TITLE
Run7 Chamber

### PR DESCRIPTION
**Pull request by Jchhokar**
Addition of Run7 small chamber geometry. Chamber is made from plastic material (polyamide PA6) and in the centre of the chamber there is a filing of XAD4 (thickness 6mm).

### CODE REVIEW

**Comment by JPETTomographyTester** [comment]
Can one of the admins verify this patch?

File: src/DetectorConstruction.cc [hunk 1]
```diff
@@ -98,9 +102,59 @@ void DetectorConstruction::LoadGeometryForRun(G4int nr)
...
+
+void DetectorConstruction::ConstructTargetRun7()
+{
+	G4RotationMatrix rot = G4RotationMatrix();
+
+   G4double z[] = {-8.8*cm, -7.8*cm, -7.8*cm, -6.6*cm, -6.6*cm, -5.1*cm, -4.9*cm, -3.1*cm, -2.8*cm, 2.8*cm, 3.1*cm, 4.9*cm, 5.1*cm, 6.6*cm, 6.6*cm, 7.8*cm, 7.8*cm, 8.8*cm }; 
+   G4double rInner[] = { 6.0*mm, 6.0*mm, 6.0*mm, 5.0*mm, 5.0*mm, 5.0*mm, 5.0*mm , 5.0*mm,  5.0*mm, 5.0*mm, 5.0*mm, 5.0*mm, 5.0*mm, 5.0*mm, 5.0*mm, 6.0*mm, 6.0*mm , 6.0*mm};
+   G4double rOuter[] = { 17.5*mm, 17.5*mm, 13.5*mm, 13.5*mm,  10.0*mm, 10.0*mm, 10.0*mm, 10.0*mm, 5.0*mm, 5.0*mm, 10.0*mm, 10.0*mm, 10.0*mm, 10.0*mm, 13.5*mm, 13.5*mm, 17.5*mm, 17.5*mm }; 
+
+   G4Polycone* smallChamber = new G4Polycone("bigChamber",0*degree,360*degree, 18 , z, rInner, rOuter);
```
Comment (DariaKisielewska) [inline src/DetectorConstruction.cc]: bigChamber name is misleading here

Comment (DariaKisielewska) [reply src/DetectorConstruction.cc]: also while running code a following exception occurs:

-------- WWWW ------- G4Exception-START -------- WWWW -------
*** G4Exception : GeomMgt1002
      issued by : G4Polycone::CalculateExtent()
Triangulation of RZ contour has failed for solid: bigChamber !
Extent has been calculated using boundary box
*** This is just a warning message. ***
-------- WWWW -------- G4Exception-END --------- WWWW -------

please check if size of rOuter and rInner ... are as they suppose to be

**Review by DariaKisielewska — CHANGES_REQUESTED** [review]
General comment: are the implemented dimensions corresponds to the measured ones? 
We already had a bad experiences with technical drawings ...

File: src/DetectorConstruction.cc [hunk 2]
```diff
@@ -98,9 +102,63 @@ void DetectorConstruction::LoadGeometryForRun(G4int nr)
 
 }
 
+//JC
+
+void DetectorConstruction::ConstructTargetRun7()
+{
+	G4RotationMatrix rot = G4RotationMatrix();
+
+//   G4double z[] = {-8.9*cm, -8.85*cm, -8.8*cm, -7.81*cm, -7.8*cm, -6.61*cm, -6.6*cm, -5.1*cm, -4.9*cm, -3.1*cm, -2.8*cm, 2.8*cm, 3.1*cm, 4.9*cm, 5.1*cm, 6.6*cm, 6.61*cm, 7.8*cm, 7.81*cm, 8.8*cm, 8.85*cm , 8.9*cm};
```
Comment (DariaKisielewska) [inline src/DetectorConstruction.cc]: if those values are not needed then please remove them

File: src/DetectorConstruction.cc [hunk 3]
```diff
@@ -339,6 +398,12 @@ void DetectorConstruction::InitializeMaterials()
     smallChamberMaterial->AllowsAnnihilations(true); 
     smallChamberMaterial->Set3gProbability(foPsProbabilityAl); 
     smallChamberMaterial->SetoPsLifetime(fTauoPsAl); 
+//Jc
```
Comment (DariaKisielewska) [inline src/DetectorConstruction.cc:395]: please remove commented text

File: src/DetectorConstruction.cc [hunk 4]
```diff
@@ -98,9 +102,63 @@ void DetectorConstruction::LoadGeometryForRun(G4int nr)
 
 }
 
+//JC
```
Comment (DariaKisielewska) [inline src/DetectorConstruction.cc]: please remove commented text

**Review by DariaKisielewska — CHANGES_REQUESTED** [review]
Runt7 branch should be updated to current develop branch

**Comment by DariaKisielewska** [comment]
Merged by hand

**Pull request closed without merging**